cfme/tests/cloud/test_mgmt_sys_azure.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+import cfme.web_ui.flash as flash
+import utils.error as error
+from cfme import Credential
+from utils import testgen, version
+from utils.update import update
+from utils.mgmt_system import azuresms
+
```

Comment:
In here, we'll need to use testgen to create the parametrization, basically we'll have something like this

```python
pytest_generate_tests = testgen.generate(testgen.provider_by_type, ['azure'],
                                         scope='module')
```

Then each of the tests would use
```python
def test_azure_vm_list(provider):
    provider.mgmt.list_vm('WS2012')
```
At least that's how it should work anyway :)